invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/page.html

```diff
@@ -34,3 +34,6 @@
   {%- endblock page_footer %}
 
 {%- endblock body_inner %}
+{%- block javascript %}
+  {{ super() }}
+{%- endblock javascript %}
```

Comment:
Minor: If we simply call super we can simply remove the block and it would be the same, no?
```suggestion
```

Reply:
Now that in the child component I'm calling `super.super()`, some tests were failing because it couldn't find javascript block in here.